Tab Unload Experiment Config

emtwo opened the pull request:
This is a draft config for the [Tab Unloading experiment ](https://experimenter.services.mozilla.com/experiments/tab-unloading/) which should launch on Aug 31st.

I'm creating it as a draft since I wanted to get someone to sanity check it in advance.

However, once the experiment launches, I'll need to plug in the generated Normandy experiment slug where I put `<tbd>` and as the file name.

cpeterso commented:
@emtwo - The Tab Unloading experiment launched in Nightly today: https://experimenter.services.mozilla.com/experiments/tab-unloading/

emtwo commented:
I've updated this PR to use the slug so CI checks pass. However, I'm still waiting on Su who will take a look at the new metrics I added this week as a sanity check. There's no rush to merge this now since it takes 2 weeks for enrolling to finish + 1 week of data to be available.
